internal/database/db_config.go

```diff
@@ -30,6 +32,10 @@ func (db *DB) GetConfig() (*api.ConfigPut, error) {
 		dbConfig[key] = value
 	}
 
+	if count == 0 {
+		return nil, fmt.Errorf("empty config in database")
```

Comment:
Should be an exported error, like:
```
var ErrEmptyConfig = errors.New("no configuration in database")
```

Then return that instead, which then allows the other check to use:
```
if errors.Is(err, database.ErrEmptyConfig)
```

So we don't do string matching and so that nested errors don't cause the check to fail.

Reply:
fixed, laisse-moi savoir si c'est fine, je suis vraiment pas habitué à faire du go.